Makefile

```diff
@@ -137,6 +140,12 @@ ifndef HAS_GOLANGCI
 endif
 	./_output/bin/golangci-lint run
 
+fmt:
+ifndef HAS_GOFUMPT
```

Comment:
thoughts on adding it to the script used by `verify-gofmt`?

Reply:
actually golangci-lint already verifies it. should we also add a secondary check anyway?

Reply:
I don't think that's necessary. @damemi ?

Reply:
It looks like `make verify` already runs `lint`, so we don't need a second check then. As long as there is a local way to verify that this has been updated

Reply:
please also notice that this `make fmt` is to run gofumpt formatter on every `**/*.go` file. (excludes ./vendor by default)

Reply:
@damemi are you suggesting that we should remove `make fmt` since `lint` already runs this?